package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okteto/react-oauth2-login",
-  "version": "1.0.3",
+  "version": "2.0.0",
```

Comment:
The question you should ask yourself: Would people using React < 18 still be able to use the generated library? So, if the answer is yes, then it should be minor because the change is backwards compatible. If no, it should be major.